docs/modules/ROOT/pages/tmail-backend/smtp-extensions/tmailValidRcptHandler.adoc

```diff
@@ -0,0 +1,3 @@
+= TMail validate reception handler
+
+TODO
```

Comment:
Todo

Reply:
Task for it https://github.com/linagora/tmail-backend/issues/1149